### ISSUE TITLE
Any interest in the sphinx-book-theme for docs?

### ISSUE THREAD

**Issue by choldgraf** [jupyter/nbclient#49]
We've been working on a simple sphinx theme for the jupyter-book rewrite that I've also found useful for other documentation sites as well (sphinx-book-theme.readthedocs.io)

I find the Alabaster theme to be a little bit cluttered and hard to parse sometimes...would folks be interested in a PR that updates the docs theme for nbclient to the sphinx-book-theme? Here's an example of what it looks like by just swapping the theme name in `conf.py`:

![image](https://user-images.githubusercontent.com/1839645/81321450-76e4b800-9047-11ea-80fe-f28f4b5bc835.png)

**Comment by MSeal** [comment]
I think that looks nicer. Is there anything unsupported with the new theme or concerns around swapping in the technical sense? @willingc might have some input if we wanted to switch jupyter doc themes or not.

**Comment by choldgraf** [comment]
I think these are the things we'd need to switch:

* badges to the left that are auto-populated would be moved to the `index.rst` page rather than auto-generated
* I can't think of anything else...

We could also customize things etc, that screenshot was just the minimal change to get the theme building with these docs. The sphinx-book-theme is basically a simplified version of [the pydata sphinx theme](https://pydata-sphinx-theme.readthedocs.io/) which we've started using across the JupyterHub repositories. I've just been using the sphinx-book-theme for docs that are a bit simpler in nature

**Comment by willingc** [comment]
+1 @choldgraf to dumping alabster for the pydata based book theme. As long at it renders everything fine which it should. Only thought would be to update the sidebar to be a bit more accessible (seems to have low contrast).

**Comment by choldgraf** [comment]
ok - I will make a PR and folks can give their thoughts there. That is a good point about a11y @willingc - currently the navigation bar is low-contrast until the mouse hovers over it (or the focus is on an element inside) and then it becomes high-contrast. I don't think that's ideal for screen readers though :-/

**Comment by MSeal** [comment]
Thanks @choldgraf !
